packages/lisk-transactions/src/transactions/base.ts

```diff
@@ -101,7 +98,7 @@ export abstract class BaseTransaction {
 		this.receivedAt = rawTransaction.receivedAt;
 	}
 
-	public abstract assetToJSON(): TransactionAsset;
+	public abstract assetToJSON(): object;
```

Comment:
Why is it no longer of type TransactionAsset?

Reply:
Since asset type is defined in each child transaction, we can't expect it to be specific type.
Maybe we can use generics here, but it's only used in this class, so it wont make much sense?